packages/cloud-cognitive/src/components/FilterSummary/FilterSummary.stories.js

```diff
@@ -0,0 +1,79 @@
+//cspell: disable
```

Comment:
Probably better to disable cspell by words or lines or just around the particular lines in question rather than disabling for the whole file.

```
// cspell:disable
....
// cspell:enbable
```